### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by kathytbui**
### New Feature Submissions:
* [X] Have you added an explanation of what your changes do?
* [X] Have you linted your code locally prior to submission?
Search controller index method is still too long

### Testing:
* [X] Have you written new tests for your core changes, as applicable?
* [X] Have you successfully ran tests with your changes locally?
* [X] All new and existing tests passed?

**Test coverage** = 283/283 100%
--------------------------------------------------------------------------------
### Change Log:
- Added readme: still missing image references

### CODE REVIEW

**Review by sciencefixion — APPROVED** [review]
Gorgeous!

File: README.md [hunk 1]
```diff
@@ -1,24 +1,59 @@
...
-* ...
+[Kathy Bui's Github](https://github.com/Kathybui732) &
+[Linkedin](https://www.linkedin.com/in/kathy-bui-87a27a1ab/)
+
+[Dorion's GitHub]() &
+[Linkedin]()
+
+## Our App
+
+### Schema
```
Comment (janegreene) [inline README.md:46]: https://dbdiagram.io/home we can use this once we get the users and images related.

**Review by janegreene — APPROVED** [review]
Nice! Can't wait to add picture of the finished product too!

File: README.md [hunk 2]
```diff
@@ -1,24 +1,59 @@
...
+![Schema]()
+
+### Welcome page
+![Welcome Page](https://i.imgur.com/Bz6sJzN.png)
+
+### User Dashboard
+![User Dashboard]()
```
Comment (dtsehl) [inline README.md:53]: These can be added at a later date.

**Review by dtsehl — APPROVED** [review]

**Pull request merged**